src/modules/home/components/accommodation-card/accommodation-card.tsx

```diff
@@ -0,0 +1,44 @@
+import CardActions from '@mui/material/CardActions';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import FavoriteBorderIcon from '@mui/icons-material/FavoriteBorder';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import IconButton from '@mui/material/IconButton';
+import StarRate from '@mui/icons-material/StarRate';
+import CardImage from '#/assets/images/The-Balmoral-Executive-View-Room-e1682260891619.jpg';
+import { useState } from 'react';
+import { truncateString } from '#/utils/truncate-string.util';
+import { styles } from './styles';
+import { pink } from '@mui/material/colors';
+
+export function AccommodationCard() {
```

Comment:
Please add the support for props. This component should accept the props and render them accordingly